lib/elixir/lib/calendar.ex

```diff
@@ -171,6 +171,7 @@ defmodule Date do
 
   """
   @spec leap_year?(Calendar.date) :: boolean()
+  def leap_year?(date)
```

Comment:
I believe we usually separate bodiless definition by newline (as you have already done in few places in the PR). :bowtie: